src/Microsoft.Health.Fhir.Shared.Core/Features/Search/Parameters/SearchParameterValidator.cs

```diff
@@ -64,6 +68,7 @@ public async Task ValidateSearchParameterInput(SearchParameter searchParam, stri
                 (var activeReindexJobs, var reindexJobId) = await fhirOperationDataStore.Value.CheckActiveReindexJobsAsync(cancellationToken);
                 if (activeReindexJobs)
                 {
+                    _logger.LogInformation("Reindex job {ReindexJobId} is running. Changes to search parameters are not allowed.", reindexJobId);
```

Comment:
Just curious, but do we need this one? The JobConflictException has the reindexJobId in the message text.

Reply:
We can skip this one! Will delete